src/set_theory/surreal.lean

```diff
@@ -66,6 +66,123 @@ end
 
 instance : has_mul pgame := ⟨mul⟩
 
+/-- An explicit description of the moves for Left in `x * y`. -/
+def left_moves_mul {x y : pgame} : (x * y).left_moves
```

Comment:
Why not make `x` and `y` explicit arguments here?

Reply:
Done! Thanks,